com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ApplicationDigestDetector.java

```diff
@@ -0,0 +1,26 @@
+package com.sap.cloud.lm.sl.cf.process.util;
+
+import org.cloudfoundry.client.lib.CloudControllerClient;
+import org.cloudfoundry.client.lib.domain.CloudApplication;
+
+import com.sap.cloud.lm.sl.cf.core.helpers.ApplicationFileDigestDetector;
+
+public class ApplicationDigestDetector {
+    private CloudApplication app;
+    private CloudControllerClient client;
+
+    public ApplicationDigestDetector(CloudApplication app, CloudControllerClient client) {
+        this.app = app;
+        this.client = client;
+    }
+
+    public String getExistingAppDigest() {
```

Comment:
Keep consistency rename to getExistingApplicationDigest

Reply:
fixed